sdk/formrecognizer/ai-form-recognizer/test/node/formtrainingclient.spec.ts

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+import fs from "fs-extra";
+import path from "path";
+
+// import { getTrainingContainerSasUrl } from "../util/trainingContainer";
+import { FormRecognizerClient, AzureKeyCredential } from '../../src';
+import { env } from "@azure/test-utils-recorder";
+// import { URLBuilder } from '@azure/core-http';
+
+const ASSET_PATH = path.resolve(path.join(process.cwd(), "test-assets"));
+const recognizerClient = new FormRecognizerClient(env.ENDPOINT, new AzureKeyCredential(env.FORM_RECOGNIZER_API_KEY));
+let unlabeledModelId: string | undefined;
+let labeledModelId: string | undefined;
+
+describe("FormTrainingClient NodeJS only", () => {
+  const trainingClient = recognizerClient.getFormTrainingClient();
+
+  it("trains model with forms and no labels", async () => {
+    // const containerSasUrl = getTrainingContainerSasUrl();
+    // assert.ok(containerSasUrl, "Expect valid container sas url");
+    const containerSasUrl = "<get the url to in the container>";
```

Comment:
Need to fix error where tsc thinks `getTrainingContainerSasUrl` has the setter's signature

Reply:
Fixed by will.  Could be related to typing for arrays.